zsh/QUICKSTART.md

```
@@ -0,0 +1,407 @@
+# 🚀 Zsh Configuration Quick Start Guide
+
+## 📋 Table of Contents
+
+- [Installation](#installation)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)